diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -303,10 +306,23 @@ fun ASTNode?.isAccessibleOutside(): Boolean =
             true
         }
 
+fun ASTNode.hasSuppress(warningName: String): Boolean {
+    return parent({ node ->
+        node.findChildByType(MODIFIER_LIST)
+                ?.findAllNodesWithSpecificType(ANNOTATION_ENTRY)
+                ?.any {
+                    (it.psi as KtAnnotationEntry).shortName.toString() == Suppress::class.simpleName
+                            && (it.psi as KtAnnotationEntry).valueArgumentList?.arguments?.isNotEmpty() ?: true
```

Comment:
Why `?: true`? Does it mean that annotations without parameters will make this method return `true`?